src/subscription_manager/gui/managergui.py

```diff
@@ -523,7 +523,7 @@ def _getting_started_item_clicked(self, widget):
         try:
             # unfortunately, Gtk.show_uri does not work in RHEL 5
             DEVNULL = open(os.devnull, 'w')
-            subprocess.call(["gnome-open", "ghelp:subscription-manager"],
+            subprocess.call(["xdg-open", "ghelp:subscription-manager"],
```

Comment:
instead of doing this, I think we should use

```python
ga_Gtk.show_uri(None, 'ghelp:subscription-manager', time.time())
```
because that was what was desired originally:

> `# unfortunately, Gtk.show_uri does not work in RHEL 5`

Also, on my test RHEL6 VM this PR as-is fails because xdg-open is not installed:

```shell
[root@rhel6 ~]# xdg-open
-bash: xdg-open: command not found

[root@rhel6 ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 6.7 (Santiago)
```